api-compact.yaml

```diff
@@ -10,6 +10,8 @@
 #   /enterprise/api/v1/catalogs/
 #   /enterprise/api/v1/catalogs/{id}/
 #   /enterprise/api/v1/catalogs/{id}/courses/
+#   /enterprise/api/v1/programs/
+#   /enterprise/api/v1/programs/{uuid}/
```

Comment:
Yep, I did this as part of the old requirements. :+1: Of course, we will do `programs` here as noted on the ticket.